helm/conjur-app-deploy/charts/app-secretless-broker/templates/conjur_authn_configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.conjur.authnConfigMap.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: {{ .Values.conjur.authnConfigMap.name }}
+    labels:
+      release: {{ .Release.Name }}
+      heritage: {{ .Release.Service }}
+      app.kubernetes.io/name: "conjur-authn-configmap"
+      app.kubernetes.io/component: "conjur-authn-config"
+      app.kubernetes.io/instance: "conjur-{{ .Chart.Name }}-configmap"
+      app.kubernetes.io/part-of: "conjur-app-config"
+      conjur.org/name: "conjur-authn-configmap"
+      helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+data:
+    # authn-k8s Configuration 
+    conjurAuthnLogin: {{ required "A valid conjur.authnLogin is required!" .Values.conjur.authnLogin }}
```

Comment:
Just a thought... I think you can consolidate a bit if you convert the ConfigMap key `conjurAuthnLogin` to the target environment variable `CONJUR_AUTHN_LOGIN`, e.g.:
```
    CONJUR_AUTHN_LOGIN: {{ required "A valid conjur.authnLogin is required!" .Values.conjur.authnLogin }}
```
After making that change here, then you can use an `envFrom` entry in the `test_app_secretless_broker.yaml` file below for this ConfigMap. (You won't have to explicitly mention the `CONJUR_AUTHN_LOGIN` key, since `envFrom` says to import every field in the ConfigMap as an environment variable, no enumeration needed).

Reply:
I can apply this change to the other subcharts too, for consistency.